cvat/apps/onepanelio/views.py

```diff
@@ -39,3 +95,248 @@ def get_workflow_templates(request):
         return JsonResponse(api_response.to_dict())
     except ApiException as e:
         print("Exception when calling WorkflowTemplateServiceApi->list_workflow_templates: %s\n" % e)
+
+
+
+@api_view(['POST'])
+def get_workflow_parameters(request):
+    """This function should return a list/dict of parameters for selected workflow.
+    Additionally, use default values to pre-populate fields.
+
+    """
+    # read workflow_uid and workflow_version from request payload
+
+
+    configuration = onepanel_authorize()
+    # Enter a context with an instance of the API client
+    with onepanel.core.api.ApiClient(configuration) as api_client:
+        # Create an instance of the API class
+        api_instance = onepanel.core.api.WorkflowTemplateServiceApi(api_client)
+        namespace = os.getenv('ONEPANEL_RESOURCE_NAMESPACE')  # str |
+    try:
+        api_response = api_instance.get_workflow_template2(namespace, uid=workflow_uid, version=str(workflow_version))
+        pprint(api_response)
```

Comment:
This should be removed once development is done.
The pprint()

Reply:
yes, removed.